src/RadioButtons/index.stories.js

```diff
@@ -84,6 +84,53 @@ FullyControlled.parameters = {
   },
 };
 
+export const AsRadioButtonsWithDetails = Template.bind({});
+AsRadioButtonsWithDetails.args = {
+  options: {
+    OptionA: {
+      value: "A",
+      details:
+        "Option A details. The Option A details are very long and will wrap to the next line. The Option A details are very long and will wrap to the next line.",
+    },
+    OptionB: { value: "B", details: "Option B details" },
+    OptionC: { value: "C", details: "Option C details" },
+  },
+  name: "card_options_with_details",
+};
```

Comment:
Is this a breaking change for existing RadioButtons usage?

If so, maybe we could set up a separate `optionsDetails` prop or something like that?

Reply:
I missed that, good call!